smac/utils/multi_objective.py

```diff
@@ -0,0 +1,30 @@
+from typing import Union, List, Tuple
+import numpy as np
+
+
+def normalize_costs(values: Union[np.ndarray, list],
+                    bounds: List[Tuple[float, float]] = None) -> np.ndarray:
+
+    if isinstance(values, list):
+        values = np.array(values)
+
+    normalized_values = np.empty_like(values)
+    for axis in range(values.shape[1]):
```

Comment:
Seems off to me, does `values.shape[1]` really contain all the axes. Maybe it's meant to be `column`?

```python
values = np.array([[1,2,3],[4,5,6]])
values.shape[1] == 3  # Yet there's only two axes?
list(range(values.shape[1]))) # [0,1,2]
```